src/ocean/util/app/ext/LogExt.d

```diff
@@ -79,22 +79,39 @@ class LogExt : IConfigExtExtension
     public bool use_insert_appender;
 
 
+    /// If set, will be used by `makeLayout` to map a name to a `Layout`
+    private Appender.Layout delegate (cstring name) layout_maker;
+
+
     /***************************************************************************
 
         Constructor.
 
         Params:
+            make_layout         = A delegate that instantiates an
+                                  `Appender.Layout` from a name. If null,
+                                  defaults to `ocean.util.Config: newLayout`.
             use_insert_appender = true if the InsertConsole appender should be
                                   used (needed when using the AppStatus module)
 
     ***************************************************************************/
 
-    this ( bool use_insert_appender = false )
+    this ( bool use_insert_appender )
     {
-        this.use_insert_appender = use_insert_appender;
+        this(null, use_insert_appender);
     }
 
 
+    /// Ditto
```

Comment:
You might not use Ditto here (to be consistent with other similar situations in other functions). So new documentation needs to be added to this constructor and the parameter `make_layout`  should be removed from the constructor above